rust/src/crypto.rs

```diff
@@ -78,31 +78,17 @@ impl CryptoCurveUtils {
     }
 }
 
-/// This method encodes raw bytes as CBOR, tagging the payload as
-/// a Tag24 data item and constructing a COSE_Sign1 object that is
-/// signed by the signing key, with the signature included in the
-/// CBOR bytes encoded COSE_Sign1 object returned.
+/// This method accepts raw bytes to signed and included in a
```

Comment:
NOTE: Will need to run the uniffi bindings again to ensure the comments are correctly added to the swift bindings.

```suggestion
/// This method accepts raw bytes to be signed and included in a
```